### ISSUE TITLE
Make it clearer which tab is selected in search modal

### ISSUE THREAD

**Issue by lucacasonato** [denoland/dotland#2354]
Currently the selected item is just highlighed very subtly using a slightly different text color. There should be a clearer distinction, for example by having an underline on the selected tab.

**Comment by crowlKats** [comment]
There is an underline though.

**Comment by lucacasonato** [comment]
There is no underline:
![image](https://user-images.githubusercontent.com/7829205/183418537-3469e24a-f363-484d-bc74-27031f80ea9a.png)

**Comment by crowlKats** [comment]
interesting:
<img width="368" alt="Screenshot 2022-08-08 at 14 32 15" src="https://user-images.githubusercontent.com/13135287/183418614-f399359d-7640-4b03-8cf6-92a4630bb3d2.png">

**Comment by crowlKats** [comment]
this seems to happen on chrome & safari (in ff it doesnt):
<img width="480" alt="Screenshot 2022-08-08 at 14 43 24" src="https://user-images.githubusercontent.com/13135287/183420730-edb3d83c-8e76-4c33-b22e-ed9e96be460a.png">

**Comment by danyalasif** [comment]
I have tested this a bit and it always gives the `INJECT_CSS_ERROR` on chrome, I was able to fix this by using border directly to give the underline and remove the `css` call.

Is this issue open for PR?
